npm/package.json

```diff
@@ -66,6 +68,7 @@
     "@types/sinon": "10.0.13",
     "@types/tap": "15.0.8",
     "cross-env": "7.0.3",
+    "prettier": "^2.8.3",
```

Comment:
`prettier` can be removed here, since we have added it as a dev dependency in the root package.json. You can run `npm run custom-install` from the main folder.